tests/sdk-consumer/app.php

```diff
@@ -6,12 +6,15 @@
 
 use \SwedbankPay\Api\Client;
 
+$version = getenv('VERSION');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function getenv() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5631375)